frontend/src/components/Uploader/ProvidersUploader.tsx

```diff
@@ -49,13 +52,13 @@ function onClearInputs(store: Store) {
  * Aliases are compared to column names as lowerCase()
  */
 const COLUMNS = [
-  { aliases: ['Address'], required: true },
-  { aliases: ['City'], required: true },
-  { aliases: ['State'], required: true },
-  { aliases: ['Postal Code', 'Zip', 'ZipCode', 'Zip Code'], required: true },
+  { aliases: ['Address'], required: (ENV !== 'PRD') },
+  { aliases: ['City'] },
```

Comment:
not required in not prod?

Reply:
required in not PRD

Reply:
Really? So we don't want to accept `latitude, longitude` alone in not PRD?

Reply:
We could also never require both and simply use the parsing checks defined below.